### PR TITLE
Create /structure, YAML folders; Minor fixes

### PR DESCRIPTION
**Pull request by jwbaker-mitre**
Various updates and edits.

- Created /structure folder for housing all structural documentation and examples
- Created APT29 /YAML folder for housing alternative community produced YAML files, similar to FIN6 plan.
- Corrected several hyperlink references, dates, and formatting issues

### CODE REVIEW

**Review by MarkDavidson — CHANGES_REQUESTED** [review]
Do you have a way to make the title more accurate while keeping it short? If not, that's OK.

File: apt29/NOTICE.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright 2020 MITRE Engenuity. Approved for public release. Document number AT0008.
+Copyright 2021 MITRE Engenuity. Approved for public release. Document number AT0008.
```
[the remark this answers is not in the document]
Comment (jwbaker-mitre) [reply apt29/NOTICE.txt:1]: Agreed on the date range, however we should ask @jonathanbaker on this one.

File: apt29/Emulation_Plan/YAML/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Machine-Readable APT29 Emulation Plans
+
```
[the remark this answers is not in the document]
Comment (jwbaker-mitre) [reply apt29/Emulation_Plan/YAML/README.md]: 1. The directory is now lowercase.
2. YAML file for each plan is now contained within the respective yaml folder.  I had to check to ensure this did not break any plugin functionality with the CALDERA team, and it does not.
3. The placeholder has now been replaced with the initial emulation plan yaml entry.

File: structure/format_dictionary.yaml [hunk 3]
```diff
@@ -2,8 +2,8 @@
 
 - emulation_plan_details:
     id: UUID for the Procedure (e.g. b5192070-dff1-4134-bf66-c6b87c7498af)
-    adversary_name: menuPass
-    adversary_description: menuPass is thought to be a Chinese cyber espionage group associated with activity directed by elements of the PRC MSS.
+    adversary_name: Adversary
```
[the remark this answers is not in the document]
Comment (jwbaker-mitre) [reply structure/format_dictionary.yaml:5]: @MarkDavidson I like the use of angle brackets, however nearly every field on this format_dictionary is a variable.  Do you think we should proceed with encapsulating all of them in brackets or leaving them as they currently are?

File: apt29/NOTICE.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (jwbaker-mitre) [reply apt29/NOTICE.txt:1]: Based on Frank's concurrence as well, I've changed the overall library to include 2021.  The individual plans still hold the copyright date of the year of their release respectively.  https://github.com/center-for-threat-informed-defense/adversary_emulation_library/pull/57/commits/4be17d4d651fe792a1dc6f3e950a18ddbc8dbeea

File: structure/format_dictionary.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (MarkDavidson) [reply structure/format_dictionary.yaml:5]: @jwbaker-mitre I'll leave it to your judgement. The result should be whatever is best for the author of a new emulation plan.

[the remark this answers is not in the document]
Comment (jwbaker-mitre) [reply structure/format_dictionary.yaml:5]: I am going to leave it in its current state and we can revisit at a later time.

**Review by MarkDavidson — APPROVED** [review]

**Pull request merged**